kbl/kbl-v24/pom.xml

```diff
@@ -90,6 +42,7 @@
                             <goal>generate</goal>
                         </goals>
                         <configuration>
+                            <locale>en</locale>
```

Comment:
Perhaps a comment should be added why this configuration is needed. Maybe also in all places.